package.json

```diff
@@ -57,5 +57,9 @@
     "playkit-js",
     "playkit-js-ivq",
     "html5 player"
+  ],
+  "kalturaPluginDependencies" : [
```

Comment:
can we make it shorter and call it "kalturaDependancies" instead? I don't think it'll always be plugins (for example base player of ovp, ott or path could also be treated as dependancies.. or in the future we may want to add non-plugin libraries, etc.)

Reply:
*kalturaDependencies (typo)